data/common.yaml

```diff
@@ -20,6 +23,13 @@ mcollective::server_config:
   plugin.yaml: "/etc/puppetlabs/mcollective/generated-facts.yaml"
   registerinterval: 10
   registration: "Choria"
+  logfile: "/var/log/puppetlabs/mcollective.log"
```

Comment:
I think its worth making this mcollective class property called `$server_logfile`, this is one other classes would want to reference and sitting it deep inside the config hash would hinder that